src/examples/HelloSPTP.cpp

```diff
@@ -138,15 +136,15 @@ void run() {
 
       const size_t timeTotal = (size_t)floor(tAll.getElapsed());
       cout << "Total elapsed time = " << timeTotal << " seconds" << endl;
-      #ifdef NDEBUG
-	    #ifdef _MSC_VER
+      if(EPOCHS >= 100) { //show only relevant values, ie don't run in valgrind (ndebug, epochs=5) run
+#ifdef _MSC_VER
           const size_t CI_avg_time = (size_t)floor(14*Timer::getSpeed()); //sec
```

Comment:
I doubled the time in order to get windows to work.  It probably does not need quite that much time but it does take longer.

Reply:
this is unrelated, just formatting from my side. I changed #IFDEF nDEBUG to if(EPOCHS >=) as to handle the valgrind, which takes ages, but runs in Release.

Reply:
I assume you are aware that there is no valgrind in MSVC so don't make a change that would break me :)

Reply:
I'll never let you donw, ... :wink: